IPython/core/crashhandler.py

```diff
@@ -53,6 +53,16 @@
 If you want to do it now, the following command will work (under Unix):
 mail -s '{app_name} Crash Report' {contact_email} < {crash_report_fname}
 
+Please also include in your email:
+- information about your operating system: is it Linux, macOS, Windows, other,
+  and which version, for example Ubuntu 16.04, macOS 10.13.2, Windows 10 Pro,
+  is it 32-bit or 64-bit, ...
```

Comment:
Ending a paragraph with `...` to signify that these are examples seems a bit too casual for the tone of this message. Let's find a way to make it read a bit better.